### PR TITLE
fix Labels in modal box

### PR DESCRIPTION
**Pull request by Elgorm**
Small Fix.

### CODE REVIEW

File: src/widgets/nestable/Nestable.php [hunk 1]
```diff
@@ -119,7 +119,7 @@ public function init()
         }
 
         /** @var ActiveRecord|TreeInterface $model */
-        $model = $this->modelClass;
+        $model = new $this->modelClass;
```
Comment (voskobovich) [inline src/widgets/nestable/Nestable.php]: Why did you create a new object?

File: src/widgets/nestable/Nestable.php [hunk 2]
```diff
@@ -348,6 +348,9 @@ private function renderForm()
     {
         /** @var ActiveRecord $model */
         $model = new $this->modelClass;
+        $newNodeString = Yii::t('vendor/voskobovich/yii2-tree-manager/widgets/nestable','New node');
```
Comment (voskobovich) [inline src/widgets/nestable/Nestable.php]: Please, rename variables as:
```
$labelNewNode
$labelCloseButton
$labelCreateNode
```

**Review by voskobovich — COMMENTED** [review]
Please make edits and I'll take this request

File: src/widgets/nestable/Nestable.php (hunk 1 above)
Comment (voskobovich) [reply src/widgets/nestable/Nestable.php]: Please give answer to this question.

Comment (Elgorm) [reply src/widgets/nestable/Nestable.php]: I made a mistake already corrected.

**Pull request merged**